### PR TITLE
Filling out `FeatureFlag` enum

### PR DESCRIPTION
**Pull request by mike12345567**
## Description
Adding all feature flags to enum, just so there is a simple reference to see all feature flags that are available.

This also updates pro back to master state.

### CODE REVIEW

**Review by samwho — APPROVED** [review]

File: packages/backend-core/src/features/features.ts [hunk 1]
```diff
@@ -267,9 +267,9 @@ export class FlagSet<V extends Flag<any>, T extends { [key: string]: V }> {
 // All of the machinery in this file is to make sure that flags have their
 // default values set correctly and their types flow through the system.
 export const flags = new FlagSet({
-  DEFAULT_VALUES: Flag.boolean(env.isDev()),
-  AUTOMATION_BRANCHING: Flag.boolean(env.isDev()),
-  SQS: Flag.boolean(true),
+  [FeatureFlag.DEFAULT_VALUES]: Flag.boolean(env.isDev()),
+  [FeatureFlag.AUTOMATION_BRANCHING]: Flag.boolean(env.isDev()),
+  [FeatureFlag.SQS]: Flag.boolean(true),
```
Comment (shogunpurple) [inline packages/backend-core/src/features/features.ts:272]: Just realised we are missing the `FeatureFlag.BUDIBASE_AI` from here - can we add it so it goes with this PR?

File: packages/types/src/sdk/featureFlag.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
 export enum FeatureFlag {
   PER_CREATOR_PER_USER_PRICE = "PER_CREATOR_PER_USER_PRICE",
   PER_CREATOR_PER_USER_PRICE_ALERT = "PER_CREATOR_PER_USER_PRICE_ALERT",
+  AUTOMATION_BRANCHING = "AUTOMATION_BRANCHING",
+  SQS = "SQS",
   AI_CUSTOM_CONFIGS = "AI_CUSTOM_CONFIGS",
+  DEFAULT_VALUES = "DEFAULT_VALUES",
```
Comment (shogunpurple) [inline packages/types/src/sdk/featureFlag.ts:7]: Here also

**Review by shogunpurple — APPROVED** [review]
Your changes LGTM, worth us just adding the missing `BUDIBASE_AI` feature flag to it that was missed previously, I can do it on your branch if that's ok

File: packages/types/src/sdk/featureFlag.ts (hunk 2 above)
Comment (mike12345567) [reply packages/types/src/sdk/featureFlag.ts:7]: Done!

File: packages/backend-core/src/features/features.ts (hunk 1 above)
Comment (mike12345567) [reply packages/backend-core/src/features/features.ts:272]: Done!

**Pull request merged**